test/xcm/statemine.transfer.test.ts

```diff
@@ -0,0 +1,44 @@
+import { BN, BN_FIVE, BN_TEN } from "@polkadot/util";
+import _ from "lodash";
+import { AssetId, ChainId } from "../../utils/ChainSpecs";
+import { waitForEvent, waitForEvents } from "../../utils/eventListeners";
+import { StatemineNode } from "../../utils/Framework/Node/StatemineNode";
+import { alice, api, setupApi, setupUsers } from "../../utils/setup";
+import { signSendSuccess } from "../../utils/sign";
+import { XToken } from "../../utils/xToken";
+
+describe("XCM transfers", () => {
+    let statemine: StatemineNode
+
+    beforeAll(async () => {
+        // devops/chopsticks/statemine.yml port
+        statemine = await StatemineNode.create("ws://127.0.0.1:9948", ChainId.Statemine)
+        await setupApi()
+        setupUsers()
+    })
+
+    // todo repeat for every other asset
```

Comment:
test.each? 
```
  test.each([address_1, address_2])(
    //, address_1 ])(
    "xyk-pallet: Create new users with bonded amounts.",
    async (address) => {
 ```

Reply:
maybe, also it is needed to modify chostick config yml for alice to have the tokens